src/styles/components/pages/my-bookings/MyBookingsList/BookingsList/BookingItem/_BookingItem.scss

```
@@ -1,4 +1,89 @@
+.mb-my-booking {
+  background-color: $white;
+  border-radius: 4px;
+  margin-bottom: rem(16px);
+
+  .isDuo {
+    padding-top: rem(20px);
+  }
+
+  .duo-logo {
+    border-radius: 8px;
+    color: $primary;
+    font-size: rem(12px);
+    font-weight: 700;
+    line-height: 1.3rem;
+    margin-right: rem(4px);
+    padding-bottom: rem(1px);
+    padding-left: rem(1px);
+    padding-right: rem(2px);
+  }
+
+  .teaser-arrow-img {
+    margin: 0;
+  }
+
+  .teaser-link {
+    height: rem(93px);
+    overflow: hidden;
+    position: relative;
+  }
+
+  .teaser-thumb {
+    width: 20%;
+  }
+
+  .teaser-wrapper {
+    width: 65%;
+  }
+}
+
+.mb-token-container {
+  @include dotted($primary, top);
+
+  overflow: hidden;
+  position: relative;
+
+  hr {
+    border: 1px solid $grey-border-mb;
+    display: inline-block;
+    height: rem(33px);
+    margin-top: rem(6px);
+    width: 1px;
+  }
+}
+
+.mb-corner {
+  border-radius: 100%;
+  height: 20px;
+  position: absolute;
+  width: 20px;
```

```suggestion
  width: rem(20px);
```
Je te laisse faire une passe sur les autres.